packages/cli/src/ceramic-daemon.ts

```diff
@@ -344,24 +344,34 @@ export class CeramicDaemon {
 
     // Now that ceramic node is set up we can start publishing metrics
     // publishing metrics is enabled by default, even if no metrics config
-    if (! opts.metrics || opts.metrics?.metricsPublisherEnabled) {
+    if (!opts.metrics || opts.metrics?.metricsPublisherEnabled) {
       const ipfsVersion = await ipfs.version()
-      NodeMetrics.start({
-        ceramic: ceramic,
-        network: params.networkOptions.name,
-        ceramicVersion: version,
-        ipfsVersion: ipfsVersion.version,
-        intervalMS: opts.metrics?.metricsPublishIntervalMS || DEFAULT_PUBLISH_INTERVAL_MS,
-        nodeId: ipfsId.publicKey, // what makes the best ID for the node?
-        nodeName: '', // daemon.hostname is not useful
-        nodeAuthDID: did.id,
-        nodeIPAddr: '', // daemon.hostname is not the external name
-        nodePeerId: ipfsId.publicKey,
-        logger: diagnosticsLogger,
-      })
-      diagnosticsLogger.imp(
-        `Publishing Node Metrics publicly to the Ceramic Network.  To learn more, including how to disable publishing, please see the NODE_METRICS.md file for your branch, e.g. https://github.com/ceramicnetwork/js-ceramic/blob/develop/docs-dev/NODE_METRICS.md`
-      )
+      let didId: string | null = null;
+      try {
+        didId = did.id
+      } catch (err: any) {
+        diagnosticsLogger.imp(
+          `Unable to publish node metrics without an authenticated DID.  If you wish your ceramic node to be able to publish metrics a privateSeedUrl is required. Error: ${err.toString()}`
```

Comment:
`privateSeedUrl` is required, the Ceramic daemon won't start up without it.  You need the node DID to authenticate requests to the CAS.  So there's no way the node could not have a DID.  I think there must be something else going on here.

Reply:
I don't know, but in fact it is not present, and if i add it, then it doesn't fail

`https://github.com/ceramicstudio/simpledeploy/blob/main/k8s/base/composedb/daemon-config.json#L25`